app/src/androidTest/java/ch/sdp/vibester/activity/ProfileActivityTest.kt

```diff
@@ -122,14 +126,37 @@ class ProfileActivityTest {
         createMockImageGetter()
 
         val scn: ActivityScenario<ProfileActivity> = ActivityScenario.launch(intent)
+        onView(withId(R.id.profile_scores)).perform(click())
+
+        onView(withId(R.id.profile_scroll_stat)).check(matches(isDisplayed()))
+
+        onView(withId(R.id.profile_top_tracks)).check(matches(withText(inputProfile.scores.getOrDefault("top tracks", 0).toString())))
+        onView(withId(R.id.profile_kpop)).check(matches(withText(inputProfile.scores.getOrDefault("kpop", 0).toString())))
+        onView(withId(R.id.profile_rock)).check(matches(withText(inputProfile.scores.getOrDefault("rock", 0).toString())))
+        onView(withId(R.id.profile_bts)).check(matches(withText(inputProfile.scores.getOrDefault("BTS", 0).toString())))
+        onView(withId(R.id.profile_imagine_dragons)).check(matches(withText(inputProfile.scores.getOrDefault("Imagine Dragons", 0).toString())))
+        onView(withId(R.id.profile_billie_eilish)).check(matches(withText(inputProfile.scores.getOrDefault("Billie Eilish", 0).toString())))
+    }
+
+    @Test
+    fun clickBackToMain(){
+        val inputProfile = User("Lalisa Bon","bit.ly/3IUnyAF", "lisa@test.com",  12, 8)
```

Comment:
It's a test class so shouldn't be an issue, but would it be possible to put links in strings.xml? It would help others with understanding where that link leads to